bundle/bundle_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
nit: I think this alias is no-op, go already uses the part before the first dot as import name.

Reply:
Yeah I'm aware it's a noop. It's an artifact of using GoLand for an IDE and for the life of me I can't figure out how to stop it from introducing those. When I remove the alias, it adds it back. 🤷‍♀  Running goimports (which I do on file save) doesn't clean those up either.

Reply:
I just asked the twitterverse for help. https://twitter.com/carolynvs/status/1169993866503757827